### DISCUSSION TITLE
Centronics 50 3D print needs an improvement

### DISCUSSION THREAD

**Discussion by badass-boss** [BlueSCSI/BlueSCSI-v2#204] [Ideas]
I 3D printed a case for BlueSCSIv2 centronics 50 version and it needs some improvements. The bottom half can get a pass but the top half is paper thin and this thing barely keeps itself together and the case opens up constantly. Top half definitely needs to be thicker and more sturdy or both halves and I think the latches should be reworked and be bigger/stronger. Another addition I would make is make the gap higher for the USB port (on the top half of the case) so if one uses a socketed Raspberry Pico they would have no problem with inserting the USB cable through the case. That would be ideal

**Comment by erichelgeson** [comment]
Thanks for the feedback - I'll say we're not 3d designers and [the community](https://www.printables.com/@EricHelgeson_1492862/collections/1024938) makes a lot of great cases (though the Centronics isn't as popular so not as many people have made cases) - If you'd like to make a case or make updates to the existing one please let us know!
